rfi/explanation/explanation.py

```diff
@@ -40,6 +40,14 @@ def __init__(self, fsoi, lss, fsoi_names, ex_name=None):
             logging.debug('lss shape: {}'.format(lss.shape))
             raise RuntimeError('Lss has incorrect shape.')
 
+    def fi_vals(self):
+        """ Computes the sample-wide RFI for each run
+        
+        Returns:
+            (#fsoi, #runs)
+        """
+        return np.mean(np.mean(self.lss, axis=3), axis=2)
```

Comment:
You can actually write np.mean(self.lss, axis=[3, 2])

Reply:
Good point! Also more consistent